### ISSUE TITLE
Pagination buttons hidden behind footer

### ISSUE THREAD

**Issue by dillonhafer** [bensheldon/good_job#533]
The pagination buttons are behind the very cool motivational footer. I love the footer with the timestamp, but it can cover the pagination links. Not a big deal, just mentioning it.

<img width="283" alt="zoomed-out" src="https://user-images.githubusercontent.com/1136388/156091495-ee2250f5-d412-4886-8e33-efb1e1f9a035.png">
<img width="370" alt="normal-zoom" src="https://user-images.githubusercontent.com/1136388/156091497-ec35886b-72b1-4b22-8c12-2ed4c7977c4f.png">

**Comment by dillonhafer** [comment]
BTW, I'm very much in love with this library! ❤️

**Comment by bensheldon** [comment]
@dillonhafer Oops, I'll fix that right now 😓 And yay 🙌 

I'd love to know what you love about GoodJob too.

**Comment by bensheldon** [comment]
I just released [`v2.11.1`](https://github.com/bensheldon/good_job/releases/tag/v2.11.1) with a fix 🎉

**Comment by dillonhafer** [comment]
It has been the only PostgreSQL background job library I could find that supports ruby 3+ from day 1; I was using que, until ruby 3 came out, then I had to switch. Also, the built-in dashboard is amazing, the part I always wished que had. I run tens of thousands of credit card payments every month with the cron functionality, and I've never worried about good_job, it has never failed. I've upgraded to ruby 3.1.1 and rails 7.0.2.2, and good_job didn't care, it just worked as normal. I can't thank you and the contributors enough for such a stable well crafted library.
